### ISSUE TITLE
Can I  train it with my own dataset?

### ISSUE THREAD

**Issue by UmirT** [cunjian/pytorch_face_landmark#11]
Hello, here i did not see train.py. How can I train with my own dataset. It is possible?

**Comment by cunjian** [comment]
Check this one here: 
https://pytorch.org/tutorials/beginner/data_loading_tutorial.html
Compared to the ImageNet classification, the difference lies in the preparation of the data loader for the landmark detection task. You can utilize any existing repository developed for ImageNet classification. Also, you have to choose the appropriate loss function.
